setup.md

```diff
@@ -177,6 +177,7 @@ iOS devices.
    $ brew install --HEAD libimobiledevice
    $ brew install ideviceinstaller ios-deploy
    $ brew install cocoapods
+   $ pod setup
```

Comment:
This line (and 179) should be moved up to a separate Pod section (the present section is only for those that deploy to physical devices)